docs/codeql/codeql-language-guides/specifying-additional-remote-flow-sources-for-javascript.rst

```diff
@@ -10,8 +10,6 @@ Specifying additional remote flow sources for JavaScript
    Specifying remote flow sources with the JSON format described here is soon to be deprecated
    and will be removed in the future.
 
-   See :ref:`Customizing Library Models for JavaScript <customizing-library-models-for-javascript>`.
-
 You can model potential sources of untrusted user input in your code without making changes to the CodeQL standard library by specifying extra remote flow sources in an external file.
```

Comment:
Does this paragraph make sense without the previous link?

Reply:
In case you're wondering how it worked, the JSON files were extracted by the JavaScript extractor from the repository being analyzed. They were not part of a qlpack and could not be distributed with a CodeQL bundle. It was purely for customisation.